MediaGalleryUi/view/adminhtml/web/js/directory/directoryTree.js

```diff
@@ -17,9 +17,11 @@ define([
             filterChipsProvider: 'componentType = filters, ns = ${ $.ns }',
             directoryTreeSelector: '#media-gallery-directory-tree',
             getDirectoryTreeUrl: 'media_gallery/directories/gettree',
+            imageName: 'media_gallery_listing.media_gallery_listing.media_gallery_columns.thumbnail_url',
             modules: {
                 directories: '${ $.name }_directories',
-                filterChips: '${ $.filterChipsProvider }'
+                filterChips: '${ $.filterChipsProvider }',
+                imageModel: '${ $.imageName }'
```

Comment:
Let's reverse the dependency and observe the selected directory in the image component.